src/atoms/identicon/Identicon.styles.ts

```diff
@@ -4,29 +4,31 @@ import { tablet } from '../../layouts/breakpoints';
 import { Styled } from '../../theme';
 import { IdenticonProps, MarkedIdenticonProps } from './types';
 
-export const Container = Styled.div<Pick<IdenticonProps, 'hasInteraction' | 'size'>>`
+export const Container = Styled.div<Pick<IdenticonProps, 'hasInteraction' | 'size' | 'isProfile'>>`
```

Comment:
what's the idea behind isProfile? Like when is that option supposed to be used? I think it might be too specific. It probably should be something like `withBorder` or something like that  to not tie it to any specific page use case

Reply:
It could be too specific @ArinCantCode , the identicon in the navbar has the blue border, the rest have a different style

Reply:
lets maybe call it just `gradientBorder?: boolean` then

Reply:
well the default version is also a gradient, but yeah I agree we should probably call this something else

Reply:
actually @ArinCantCode I made a mistake here, it is actually for the profile page only it seems https://uniwhales.atlassian.net/browse/FE-765?atlOrigin=eyJpIjoiNTc0ZjY1NjZmNTlhNDY2Mzg3MmU5ODNhNDY4MWRkOWUiLCJwIjoiaiJ9. so instead of `isProfile` maybe `isProfilePage` makes sense?

Reply:
my point is that since its part of component library in the future this might be reused in other places too. So going for generic gradientBorder boolean won't confuse us later while if we do reuse it on lets say stats dashboard you will end up with isProfilePage in the code which would be weird. Best if those components do not know on which page they are used